stepfunctions_activity_worker/activity_worker.py

```diff
@@ -39,16 +43,8 @@ def _default_config():
         config = botocore.config.Config(read_timeout=70)
         return boto3.client('stepfunctions', config=config)
 
-    def __call__(self):
```

Comment:
Removal of this is a user-space breaking change; either we'll have to extend `perform_task` to allow people to pass in a task, or we'll have to make this concurrent on the `listen()` call to `perform_task()`

```python
def _poll_for_task(self):
    task = dict()
    while not task.get("taskToken"):
        print("Polling for an activity task.")
        task = self.stepfunctions.get_activity_task(**self.activity_kwargs)
    print("Recieved a task!")
    print(json.dumps(json.loads(task["input"]), indent=4, sort_keys=True))
    return task

def perform_task(self, task=None):
    task = self._poll_for_task() if not task else task
    heartbeat = Heartbeat(...)
    try:
        print(f"Performing {self.activity_name}")
        with heartbeat:
            task_input = json.loads(task["input"])
            output = self.activity_fxn(**task_input)
    except (Exception, KeyboardInterrupt) as error:
        *_, raw_traceback = sys.exc_info()
        # TODO Figure out how to handle 

```